relay-server/src/actors/processor.rs

```diff
@@ -2695,29 +2724,38 @@ impl EnvelopeProcessorService {
             EnvelopeProcessor::RateLimitFlushBuckets(message) => {
                 self.handle_rate_limit_flush_buckets(message);
             }
+            // This message is handled in the spawn_handler loop.
+            EnvelopeProcessor::UpdateGlobalConfig(_) => {}
```

Comment:
Please, add the error here. Just to avoid the problems if someone refactors the code and forgets about this piece, so we at least will catch the error and will know what happens.